packages/shared/src/components/modals/post/CommentModal.tsx

```diff
@@ -92,6 +93,11 @@ export default function CommentModal({
   const isEdit = !!editCommentId;
   const isReply = !isEdit && !!replyToCommentId;
 
+  const { comment: commentById } = useCommentById({
+    id: isEdit ? editCommentId : parentCommentId,
+    options: { enabled: !!editCommentId || !!parentCommentId },
```

Comment:
When is the modal triggered with `parentCommentId` id only?

Reply:
In case of replies. But in that case, I guess we don't need to fetch, since we should have everything already. Let me modify the code, so that it's clearer.